README.md

```diff
@@ -15,6 +15,7 @@ their functionality.
 
 ## Template Pipelines
 
+* [BigQuery to Bigtable](v2/bigquery-to-bigtable/src/main/java/com/google/cloud/teleport/v2/templates/BigQueryToBigtable.java)
```

Comment:
Wasn't sure if this should be in here since the other v2 ones aren't in here.

Reply:
It's ok to be here. We're probably going to completely rewrite the README once we start setting up proper documentation.